8/SiloHost/src/Program.cs

```diff
@@ -9,70 +8,29 @@
 using Orleans.Configuration;
 using Orleans.Hosting;
 using Orleans.Statistics;
-using System;
-using System.Net;
-using System.Net.NetworkInformation;
-using System.Net.Sockets;
 using System.Threading.Tasks;
+using Amazon.Util.Internal;
 
 namespace SiloHost
 {
     class Program
     {
+        private static readonly EnvironmentVariables environmentVariable = new EnvironmentVariables();
```

Comment:
`environmentVariable` indicates it's one variable, while it's more like a service retrieving env variables.

Reply:
And since this acts as a service, let's maybe leverage IoC for this.